eosio/contracts/tropical/tropical.contracts.md

```diff
@@ -10,3 +10,34 @@ icon: https://upload.wikimedia.org/wikipedia/commons/thumb/4/42/Love_Heart_SVG.s
 Liking this property will be visible to anyone who views your profile or searches your name.
 This like may result in the post owner’s property to be featured on the property owner’s most liked pages.
 Any usage of bots, macros, or any autonomous form of liking a specific person’s property would result in the investigation of like legitimacy for a post.
+<h1 class="contract">
+   rent
+</h1>
+---
+spec_version: 0.2.0
+title: Rent a Property
+summary: Rent a Tropical Example property
+icon: https://https://upload.wikimedia.org/wikipedia/commons/thumb/3/3e/Dollar_sign_in_circle.svg/240px-Dollar_sign_in_circle.svg.png#27F7CC5F628A8FC0B16680C141D58D42A1F49E3E38B1A4F67590C61F5D607CBF
+---
+Renting a property requires 2FA and constitutes a binding agreement to pay the listed rental fee(s).
+Any usage of bots, macros, or any autonomous form of liking a specific person’s property would result in the investigation of like legitimacy for a post.
+<h1 class="contract">
+   check2fa
+</h1>
+---
+spec_version: 0.2.0
+title: Validate the 2FA Token
+summary: Validate a WebAuthn second factor
+icon: https://upload.wikimedia.org/wikipedia/commons/thumb/d/d4/OOjs_UI_icon_key-ltr.svg/200px-OOjs_UI_icon_key-ltr.svg.png#0A1E2435DCAFDF4B34E60C65C570DCE30FAAC1862D7C5E139D188A6EAAF95014
+---
+Any usage of bots, macros, or any autonomous form of liking a specific person’s property would result in the investigation of like legitimacy for a post.
+<h1 class="contract">
+   setsrvkey
+</h1>
+---
+spec_version: 0.2.0
+title: Set the Root-of-trust
+summary: This is an administrative action that sets the respected "root of trust" key that counter signs 2FA
+icon: https://upload.wikimedia.org/wikipedia/commons/thumb/d/d4/OOjs_UI_icon_key-ltr.svg/200px-OOjs_UI_icon_key-ltr.svg.png#0A1E2435DCAFDF4B34E60C65C570DCE30FAAC1862D7C5E139D188A6EAAF95014
+---
+Any usage of bots, macros, or any autonomous form of liking a specific person’s property would result in the investigation of like legitimacy for a post.
```

Comment:
This statement is repeated verbatim 2x above.  Did you intend to modify this for each of these headings?  Seems like it might be a copy/pasta thing.  All 3 speak of liking instead of renting

Reply:
The ricardians need some work.  I'm not brushed up on the ricardian spec which may be why some icons are broken as well.